overcooked_ai_py/mdp/overcooked_env.py

```diff
@@ -33,6 +36,22 @@ def __init__(self, mdp, start_state_fn=None, horizon=MAX_HORIZON, debug=False):
         else:
             raise ValueError("Mdp should be either OvercookedGridworld instance or a generating function")
         
+        if hh_data_starts:
+            assert start_state_fn is None, "Either you have hh_data_starts, or you set a custom start_state_fn"
```

Comment:
I assume it's fine to have neither? In that case it should be `"Cannot have both hh_data_starts and a custom start_state_fn"`